docs/analyzer.md

```diff
@@ -0,0 +1,25 @@
+# Analyzer
+
+FakeItEasy provides a Roslyn analyzer to detect incorrect usages of the library,
+which cannot be prevented by the API or the compiler but will typically result in
+bugs at runtime.

```

Comment:
Grammar is hard, but I think ", which" should be "that".
As written, it sounds like _all_ incorrect usages of the library cannot be prevented by the API or the compiler, but I think the intent is for the analzyer to detect those incorrect usages that cannot be prevented by the API or compiler.

(If I'm not wrong, the second part of the sentence is a [nonrestrictive clause](http://www.quickanddirtytips.com/education/grammar/which-versus-that-0), and we want a restrictive one here.)

Perhaps I've misinterpreted your intent.